pkg/xray/resource/resource_xray_repository_config_test.go

```diff
@@ -454,14 +454,11 @@ func TestAccRepositoryConfig_RepoConfigCreate_no_exposure(t *testing.T) {
 		t.Skipf("Env var JFROG_JAS_DISABLED is set to 'true'")
 	}
 
-	packageTypes := []string{"alpine", "bower", "composer", "conan", "conda", "debian", "gems", "go", "gradle", "ivy", "rpm", "sbt"}
```

Comment:
Packages : alpine, debian, gems,go , gradle, rpm now Advanced JAS is supported with exposure